utils/generateErrorfile.py

```diff
@@ -116,34 +116,38 @@ def genCHeaderFile(errors):
   return header
 
 
-# define some globals 
+# define some globals
 prologue = "/// auto-generated file generated from errors.dat\n"\
          + "\n"
-  
+
 if len(sys.argv) < 3:
-  print >> sys.stderr, "usage: %s <sourcefile> <outfile>" % sys.argv[0]
-  sys.exit()
+  print("usage: {} <sourcefile> <outfile>".format(sys.argv[0]), file=sys.stderr)
```

Comment:
do we need `\n` here?

Reply:
`print()` has an implicit newline.

Reply:
I think you confuse it with `sys.stderr.write()`